pyogrio/tests/test_geopandas_io.py

```diff
@@ -1446,6 +1446,7 @@ def test_write_nullable_dtypes(tmp_path):
     expected["col3"] = expected["col3"].astype("float32")
     expected["col4"] = expected["col4"].astype("float64")
     expected["col5"] = expected["col5"].astype(object)
+    expected.loc[1, "col5"] = None  # pandas 3.0.0+ converts to pd.NA on line above
```

Comment:
```suggestion
    expected.loc[1, "col5"] = None  # pandas converts to pd.NA on line above
```

This is not something that should have changed in 3.0dev (I think current version of pandas do the same), but what did change is that the assert function became more strict in the exact missing value (None or pd.NA in an object dtype column are no longer considered as equal)